import "MessageBase.proto";
package Protocol;
message RobotLoginReq
{
	required string verify_key = 1;
}